lambda/common.py

```diff
@@ -93,6 +99,37 @@ def get_org_account_details():
         else:
             raise
 
+def get_cache_secret(secret_arn):
```

Comment:
What's happening here?

Reply:
I guess my confusion is - this is not the normal way to get a secret and shared in the AWS console.

Reply:
see the links above IRT the layer - this should cache the cred in-memory to reduce calls to SM. I like this design for cost and speed reduction - but I can go the route of calling SM directly, just let me know

Reply:
I think I'd prefer the standard method to fetch the slack secret at execution environment creation.